File: Rinnerdos-cs50-problems-2020-x-mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Ask user for input, repeat if not in correct range (1-8)
    int height;
    do
    {
        height = get_int("Height: \n");
    }
    while (height <= 0 || height > 8); 
    // Now print the pyramid of hashes
    for (int i = 1;  i <= height; i++)
    {
        // First print the empty spaces, opposite of amount of hashes (amount of spaces = height - amount of hashes)
        for (int k = height; k > i; k--)
        {
            printf(" ");
        }

        // Then print the hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        // Print the double space in between hashes
        printf("  ");
        // Then print the hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        // End each row with a newline
        printf("\n");
    }
}

File: Rinnerdos-cs50-problems-2020-x-sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    h = get_int("Input height of the pyramids.\n")
    if (0 < h < 9):
        break

for i in range(1, h+1):
    for j in range(h-i, 0, -1):
        print(" ", end="")
    for k in range(0, i):
        print("#", end="")
    print("  ", end="")
    for k in range(0, i):
        print("#", end="")
    print()
